templates/default.json

```diff
@@ -1,5 +1,18 @@
 {
-  "dependencies": ["next@^9.2.0", "react@^16.12.0", "react-dom@^16.12.0", "prop-types", "dotenv"],
+  "dependencies": [
+    "next@^9.2.0",
+    "react",
```

Comment:
Please respect the min versions of react and react-dom that were defined here. Those are because there are new features in NextJS 9.2 that we are taking advantage of.